Dockerfile

```
@@ -1,7 +1,11 @@
 FROM gradle:7.4-jdk8-focal AS build
 WORKDIR /notification-plugin
 COPY . ./
-RUN gradle clean classes
+RUN gradle clean jar
+
+FROM eclipse-temurin:8-jre
+WORKDIR /notification-plugin
+COPY --from=build /notification-plugin/artemis-notification-plugin-*.jar /notification-plugin/artemis-notification-plugin.jar
```

```suggestion
COPY --from=build /notification-plugin/build/libs/artemis-notification-plugin-*.jar /notification-plugin/artemis-notification-plugin.jar
```
I need to change the path when building locally. The CI pipeline seems to work as-is, though.